nautobot/utilities/filters.py

```diff
@@ -13,6 +13,8 @@
 from django_filters.constants import EMPTY_VALUES
 from django_filters.utils import get_model_field, resolve_field
 
```

Comment:
Style nit: While you're in here can you please remove those extraneous newlines on lines 11 and 15?